srcpkgs/nwg-look/template

```diff
@@ -1,28 +1,24 @@
 # Template file for 'nwg-look'
 pkgname=nwg-look
-version=0.2.7
-revision=2
+version=1.0.2
+revision=3
 build_style=go
 go_import_path=github.com/nwg-piotr/nwg-look
 hostmakedepends="pkg-config"
 makedepends="cairo-devel gtk+3-devel libglib-devel pango-devel"
 depends="xcur2png"
 short_desc="GTK3 settings editor adapted to work in the wlroots environment"
-maintainer="cinerea0 <cinerea0@protonmail.com>"
+maintainer="bai <bai@baturax.com>"
 license="MIT"
 homepage="https://github.com/nwg-piotr/nwg-look"
 changelog="https://github.com/nwg-piotr/nwg-look/releases"
 distfiles="https://github.com/nwg-piotr/nwg-look/archive/refs/tags/v${version}.tar.gz"
-checksum=20a7773b1bae7ae013efa359e73dcefc89bfb516d45bba4375f59c8a9216c9c3
+checksum=fc73b3f8a55c8b3a77853694f31be851422943692d495ec80ba895e9cd4b4cfd
 
-post_install() {
-	vinstall stuff/main.glade 644 /usr/share/nwg-look
-	vinstall stuff/nwg-look.desktop 644 /usr/share/applications
-	vinstall stuff/nwg-look.svg 644 /usr/share/pixmaps
-
-	for lang in langs/*.json; do
-		vinstall ${lang} 644 /usr/share/nwg-look/langs
-	done
+do_build() {
+   make build
```

Comment:
This breaks compiling with multiple jobs `-j`, which the do_build in the go build style implements (and `make build` just calls `go build` anyway).